test/github-trigger

```diff
@@ -32,17 +27,24 @@ def main():
 
     revision = pull['head']['sha']
     statuses = github.statuses(revision)
-    status = statuses.get(opts.context, { })
-    if status.get("state", "empty") not in ["empty", "error", "failure"]:
-        if opts.force:
-            sys.stderr.write("Pull request isn't in error state, but forcing update.\n")
-        else:
-            sys.stderr.write("Pull request isn't in error state. Status is: '{0}'\n".format(status["state"]))
-            return 1
-
-    changes = { "state": "pending", "description": testinfra.NOT_TESTED, "context": opts.context }
-    github.post("statuses/" + revision, changes)
-    return 0
+    if opts.context:
+        contexts = [ opts.contexts ]
+    else:
+        contexts = list(set(statuses.keys()) & set(testinfra.DEFAULT_VERIFY.keys()))
+
+    ret = 0
+    for context in contexts:
+        status = statuses.get(context, { })
+        if status.get("state", "unknown") not in ["empty", "error", "failure"]:
```

Comment:
Default is "unkown" and we check for "empty" - do we always get an "empty" even if the context doesn't exist?
Triggering a new context should work without force.